pkg/config/config.go

```diff
@@ -28,8 +28,8 @@ import (
 var fVersion = flag.Bool("version", false, "Print version")
 var fInstance = flag.String("instance", "0", "Running instance identifier")
 var fEnvironment = flag.String("environment", "test", "(test, stage, prod)")
-var fAddr = flag.String("addr", "127.0.0.1:8988", "Bind web application port")
-var fMgmtAddr = flag.String("mgmtAddr", "127.0.0.1:8987", "Bind management port")
+var fAddr = flag.String("addr", "0.0.0.0:8988", "Bind web application port")
+var fMgmtAddr = flag.String("mgmtAddr", "0.0.0.0:8987", "Bind management port")
```

Comment:
I do not want the management port being exposed by default.  This opens too many doors and isn't as controlled as the main port.

Reply:
```suggestion
var fMgmtAddr = flag.String("mgmtAddr", "127.0.0.1:8987", "Bind management port")
```